dask/array/tests/test_array_core.py

```diff
@@ -51,6 +51,7 @@
     stack,
     store,
 )
+from dask.array.numpy_compat import _numpy_200
```

Comment:
OT: We may want to align naming conventions between things like `_numpy_xyz` and `PANDAS_GE_XYZ`.